api/src/main/java/org/ehcache/spi/loaderwriter/WriteBehindConfiguration.java

```diff
@@ -23,70 +23,80 @@
  */
 public interface WriteBehindConfiguration extends ServiceConfiguration<WriteBehindDecoratorLoaderWriterProvider> {
   /**
-   * the minimum number of seconds to wait before writing behind
+   * The minimum number of seconds to wait before writing behind.
+   *
+   * Lower than or equal to {@see getMaxWriteDelay}.
    *
    * @return Retrieves the minimum number of seconds to wait before writing behind
    */
   int getMinWriteDelay();
 
   /**
-   * the maximum number of seconds to wait before writing behind
+   * The maximum number of seconds to wait before writing behind.
+   *
+   * Greater than or equal to {@see getMinWriteDelay}
    *
    * @return Retrieves the maximum number of seconds to wait before writing behind
    */
   int getMaxWriteDelay();
 
   /**
-   * the maximum number of write operations to allow per second.
+   * The maximum number of write operations to allow per second.
+   *
+   * {@code 0} means no limit.
```

Comment:
Right, it an int, let's make this Integer.MAX_VALUE (which most probably is more than large enough even if considered a "strict" limit)